index.html

```diff
@@ -6,32 +6,48 @@
     <title>Cocktail Builder</title>
```

Comment:
On load of the page, if there are no available cocktails for me to mix, the headline ("Cocktails to Make") shouldn't show up.